server/database/queries/profile.js

```diff
@@ -0,0 +1,11 @@
+const db_connection = require('../connection');
+
+const getUserAccounts = (userId) => {
+	const sql = {
+		text: 'SELECT * FROM account WHERE user_id=$1 ;',
+		value: [`${userId}`],
```

Comment:
in the query, we can put the variable without quotes

Reply:
Fixed